sec_certs/sample/cve.py

```diff
@@ -66,11 +67,18 @@ def from_nist_dict(cls, dct: Dict[str, Any]) -> "CVE.Impact":
     ]
 
     def __init__(
-        self, cve_id: str, vulnerable_cpes: List[CPE], impact: Impact, published_date: str, cwe_ids: Optional[Set[str]]
+        self,
+        cve_id: str,
+        vulnerable_cpes: List[CPE],
+        vulnerable_cpe_configurations: list[CPEConfiguration],
+        impact: Impact,
+        published_date: str,
+        cwe_ids: Optional[Set[str]],
     ):
         super().__init__()
         self.cve_id = cve_id
         self.vulnerable_cpes = vulnerable_cpes
+        self.vulnerable_cpe_configurations = vulnerable_cpe_configurations
```

Comment:
This is exactly the schizophreny I'm referring to. We have two objects: `vulnerable_cpes` and `vulnerable_cpe_configuraitons`. We could wrap all `CPE`s into `CPEConfiguration` I suppose.